.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
please copy the properly formatted file from: https://github.com/digitalbazaar/eslint-config-digitalbazaar/tree/master/templates

Reply:
Fixed it, except I didn't add the trailing commas. Are we moving to that style now?

Reply:
trailing commas have been allowed but not required for quite some time now.  I did not know there ware any in the template file tho.